### PR TITLE
[luci] Support for requantization from int8 to uint8

### PR DESCRIPTION
**Pull request by jinevening**
This supports requantization from int8 to uint8

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to #3779 Draft PR #3823

### CODE REVIEW

File: compiler/luci/pass/src/RequantizePass.cpp [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        continue;
+
+      // Check if this was not quantized (only quantized tensors are requantized)
+      if (circle_node->quantparam() == nullptr)
+        continue;
+
+      if (_input_type == loco::DataType::S8 && _output_type == loco::DataType::U8)
+      {
+        // Check if this is not bias (bias is not requantized when int8 -> uint8)
+        auto iw = get_input_weight_of_bias(circle_node);
```
Comment (mhs4670go) [inline compiler/luci/pass/src/RequantizePass.cpp:167]: Could you explain about `get_input_weight_of_bias` function more detail? It seems there is no other place that uses this function. But then, return value doesn't seem necessary. I mean, just true, false values are enough.

Comment (jinevening) [reply compiler/luci/pass/src/RequantizePass.cpp]: You're right. The return values are not used in this pass. They are used to calculate the scales of bias in QuantizedWithMinMaxPass. I just copied the function from the source code. I've updated the code to remove the redundant codes.

Comment (jinevening) [reply compiler/luci/pass/src/RequantizePass.cpp]: Also, I've updated the code to correctly handle const tensors (not weights nor bias). PTAL.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by parjong — APPROVED** [review]

**Pull request merged**